### DISCUSSION TITLE
Problems connecting to cluster in Kubernetes

### DISCUSSION THREAD

**Discussion by Nikolaj52** [provectus/kafka-ui#2343] [Q&A] [answered]
Hi there. 
I'm running kafka with zookeeper in kubernetes and it runs fine.
I'm also trying to run kafka-ui as a pod in Kubernetes but it can't find its cluster or any brokers (got only one for now).
The UI claims the cluster is offline.
The log from the pod claims "java.lang.IllegalStateException: Error while creating AdminClient for Cluster local".
I guess it is related to the advertised listeners in Kafka, but I can't make it work.
Any ideas?

**Comment by Nikolaj52** [comment, marked as answer]
Please ignore this issue. It is a wrong kafka install that was the culprit.

**Reply by Haarolean** [reply]
glad you solved it
